src/util/path.cpp

```diff
@@ -1,12 +1,19 @@
+// POSIX headers
+extern "C" {
+#define _POSIX_C_SOURCE 2
+#include <glob.h>
+#include <sys/stat.h>
+}
+
+#include <cerrno>
+
 #include <util/scope_exit.hpp>
 #include <util/path.hpp>
 
-// POSIX header
-#include <glob.h>
-
 namespace nest {
 namespace mc {
 namespace util {
+namespace posix {
 
 std::vector<path> glob(const std::string& pattern) {
     std::vector<path> paths;
```

Comment:
Yup that's definitely fishy. I'll fix (or confirm that despite how it looks it is correct) and update. This is from a previous commit/PR.